src/plugin.js

```diff
@@ -1,6 +1,9 @@
 import videojs from 'video.js';
-import window from 'global/window';
-import {version as VERSION} from '../package.json';
+import initOverlayComponent from './overlay-component';
+
+initOverlayComponent(videojs);
```

Comment:
I would assume we want this to be initialized in the index file, not here

Reply:
That's fair. I guess we will still need a way to call `initOverlayComponent` in the plugin only scenario, so I also export that along with the plugin in the `plugin-only` bundle.